src/utils.js

```diff
@@ -1,7 +1,9 @@
 // @flow
 import get from 'lodash/get';
 
-function getData(queryProps: Object, key) {
+import type { ConnectionData } from './interfaces';
+
+function getData(queryProps: Object, key): ConnectionData {
```

Comment:
Yeah, in fact it must be the object previously returned by a query... Any suggestion to improve this? `connectionData` maybe?